docs/eth_typing.rst

```diff
@@ -49,6 +49,30 @@ Class that contains the different names used to represent hard forks on the Ethe
         Constantinople = 'Constantinople'
         Metropolis = 'Metropolis'
 
+
+ChainId
+~~~~~~~
+
+Class which contains enums for each Ethereum network chain identifier.
```

Comment:
I'm not sure "Ethereum" fits in this description. If you pulled from `ethereum-lists/chains` then it looks like, according to their documentation, they are at least all EVM compatible. So maybe we go with that verbiage.

Maybe something along these lines? :

```suggestion
`IntEnum` class defining EVM-compatible network name enums as their respective chain id `int` values.
```